### PR TITLE
Rename AWS Native Cloud Control provider

### PR DESCRIPTION
**Pull request by danielrbradley**
## Description

Ready for the GA which includes a rename, update the docs and the original launch blog post to use the new name.

Also, fix accidental reference inthe GCP docs.

Related to:
- https://github.com/pulumi/pulumi-aws-native/issues/1226

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [x] I added aliases (i.e., redirects) for all filename changes.
- [x] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/docs/using-pulumi/pulumi-packages/how-to-author.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ To get started, click the link for the boilerplate repository template that you
 
 You should name your repository (and thus, your Pulumi Package) using the following guidelines:
 
-- The name should start with `pulumi-`, like our [`pulumi-aws-native`](/registry/packages/aws-native) AWS Native Provider and our [`pulumi-eks`](/registry/packages/eks) Component for AWS Elastic Kubernetes Service (EKS)
+- The name should start with `pulumi-`, like our [`pulumi-aws-native`](/registry/packages/aws-native) AWS Cloud Control Native Provider and our [`pulumi-eks`](/registry/packages/eks) Component for AWS Elastic Kubernetes Service (EKS)
```
Comment (scottslowe) [inline themes/default/content/docs/using-pulumi/pulumi-packages/how-to-author.md:45]:
```suggestion
- The name should start with `pulumi-`, like our [`pulumi-aws-native`](/registry/packages/aws-native) AWS Native Cloud Control Provider and our [`pulumi-eks`](/registry/packages/eks) Component for AWS Elastic Kubernetes Service (EKS)
```

**Review by scottslowe — CHANGES_REQUESTED** [review]
One small typo to fix

**Review by scottslowe — APPROVED** [review]

**Review by toriancrane — APPROVED** [review]
I'm guessing Scott's approval got rolled back after he left(?)

I'm approving to unblock.

**Comment by danielrbradley** [comment]
Closing as this rename is not going ahead for now

**Pull request closed without merging**